test/rebar_uri_SUITE.erl

```diff
@@ -0,0 +1,32 @@
+-module(rebar_uri_SUITE).
+
+-export([all/0,
+         parse/1]).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("kernel/include/file.hrl").
+
+all() ->
+    [parse].
+
+parse(_Config) ->
+    #{scheme := Scheme, host := Host, path := Path} = rebar_uri:parse("https://repo.hex.pm"),
+    ?assertEqual("https", Scheme),
+    ?assertEqual("repo.hex.pm", Host),
+    ?assert(lists:member(Path, ["", "/"])),
```

Comment:
is this because the behaviour changes based on the version used?

Reply:
Correct. `uri_string:parse/1` is not a 100% equivalent to `http_uri:parse/1`.